### PR TITLE
add support to set custom map tile via config.

### PR DESCRIPTION
**Pull request by Nakan4u**
The main goal of this commit is to add a possibility to set another language for the map.
Implementation details:
https://developer.here.com/documentation/maps/topics/map-types.html (section Custom Map Styles).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 AngularJS directive for working with Nokia Here Maps
 
-### 0.1.7 latest release
+### 0.1.8 latest release
```
Comment (dverbovyi) [inline README.md]: you don't need to changes this. 
I will do it only after the release and source archiving like this https://github.com/dverbovyi/angular-heremaps/releases

File: example/app.module.js [hunk 2]
```diff
@@ -84,7 +89,7 @@
...
+                markup: './marker.svg'
```
Comment (dverbovyi) [inline example/app.module.js]: have you test this? It will give 404 for marker.svg

**Review by dverbovyi — CHANGES_REQUESTED** [review]

File: example/app.module.js (hunk 2 above)
Comment (dverbovyi) [reply example/app.module.js]: This module is used for example. The bundler doesn't process it. Please revert

File: example/marker.svg [hunk 3]
```diff
@@ -0,0 +1 @@
...wBox="0 0 40 56"><defs><filter height="200%" width="200%" y="-50%" x="-50%" id="filter-1"><feOffset result="shadowOffset1" in="SourceAlpha" dy="5" dx="0" /><feGaussianBlur result="shadowBlur1" in="shadowOffset1" stdDeviation="5" /><feColorMatrix result="shadowMatrix1" in="shadowBlur1" type="matrix" values="0 0 0 0 0   0 0 0 0 0   0 0 0 0 0   0 0 0 0.22 0" />  <feMerge><feMergeNode in="shadowMatrix1" /><feMergeNode in="SourceGraphic" /></feMerge></filter></defs><path filter="url(#filter-1)" fill="red" fill-rule="evenodd" d="M 20 48 C 20 48 4 24.48 4 15.77 C 4 7.06 11.16 0 20 0 C 28.84 0 36 7.06 36 15.77 C 36 24.48 20 48 20 48 Z M 20 6.84 C 15.01 6.84 10.97 10.83 10.97 15.75 C 10.97 20.67 15.01 24.66 20 24.66 C 24.99 24.66 29.03 20.67 29.03 15.75 C 29.03 10.83 24.99 6.84 20 6.84 Z" /><path fill="#ffffff" fill-rule="evenodd" d="M 20 7 C 24.97 7 29 11.03 29 16 C 29 20.97 24.97 25 20 25 C 15.03 25 11 20.97 11 16 C 11 11.03 15.03 7 20 7 Z" /><text font-weight="900" y="18" x="50%" font-family="Arial" font-size="8" fill="#000"><tspan text-anchor="middle" alignment-baseline="middle">SVG</tspan><tspan text-anchor="middle" y="30" x="20" alignment-baseline="middle">drag me</tspan></text></svg>
```
Comment (dverbovyi) [inline example/marker.svg:1]: please revert, the explanation in comments above. This file 404

**Review by dverbovyi — CHANGES_REQUESTED** [review]

**Comment by dverbovyi** [comment]
@Nakan4u, please apply the requested changes and solve the conflicts. After that please run `npm run build` and commit updated dist/

anyway, thanks nice job!

File: example/marker.svg (hunk 3 above)
Comment (dverbovyi) [reply example/marker.svg:1]: screen shot 2018-04-19 at 9 45 25 pm

File: README.md (hunk 1 above)
Comment (Nakan4u) [reply README.md]: ok, I'll return it back

File: example/app.module.js (hunk 2 above)
Comment (Nakan4u) [reply example/app.module.js]: Yes, my mistake in the file path. Anyway, heremaps supports file path.

File: example/marker.svg (hunk 3 above)
Comment (Nakan4u) [reply example/marker.svg:1]: Same answer as in the previous comment.

**Review by dverbovyi — APPROVED** [review]

**Pull request merged**